frontend/src/metabase/visualizations/components/ChoroplethMap.jsx

```diff
@@ -201,13 +202,13 @@ class ChoroplethMapInner extends Component {
     let projection, projectionFrame;
     // projectionFrame is the lng/lat of the top left and bottom right corners
     if (settings["map.region"] === "us_states") {
-      projection = d3.geo.albersUsa();
+      projection = d3.geoAlbersUsa();
```

Comment:
Newer versions of D3 now flattens the namespaces (e.g. `geo.albersUsa` becomes `geoAlbersUsa`)